core/src/test/java/org/jdbi/v3/core/mapper/TestEnums.java

```diff
@@ -83,7 +84,7 @@ public void testMapInvalidEnumValue() {
 
         assertThatThrownBy(() -> h.createQuery("select * from something order by id")
             .mapToBean(SomethingElse.class)
-            .findFirst()).isInstanceOf(IllegalArgumentException.class);
+            .findFirst()).isInstanceOf(UnableToProduceResultException.class);
```

Comment:
Why did this change?

Reply:
Because the new implementations throw this exception instead of a general IAE